src/topp/IDFileConverter.cpp

```diff
@@ -566,6 +584,82 @@ class TOPPIDFileConverter :
         XQuestResultXMLFile().load(in, peptide_identifications, protein_identifications);
       }
 
+      else if (in_type == FileTypes::FASTA)
+      {
+        MSExperiment exp;
+        TheoreticalSpectrumGenerator tsg;
+
+        // extract parameters and remove non tsg params
+        Param p = getParam_().copy("fasta_to_mzml:", true);
+        String enzym = p.getValue("enzym");
+        Int min_charge = p.getValue("min_charge");
+        Int max_charge = p.getValue("max_charge");
+        Int pc_charge = p.getValue("precursor_charge");
+        p.remove("enzym");
+        p.remove("min_charge");
+        p.remove("max_charge");
+        p.remove("precursor_charge");
```

Comment:
You could add max_missed_cleavages.
By the way: What is the difference to the MSSimulator when deactivating everything but MS2? Or is that not possible?

Reply:
Missed cleavages is probably a good idea.

Not sure what you mean by "deactivating everything but MS2"... Deactivate where?

Reply:
nope. And its not easy to control which peptides are ionized etc...